### DISCUSSION TITLE
Translated Obtainium Wiki (German)

### DISCUSSION THREAD

**Discussion by FreedomIsAChoice** [ImranR98/Obtainium#1649] [Show and tell]
For all native German speakers I translated the Obtainium Wiki today and add it here as a PDF.

**I hope that it will make Obtainium better known and more popular in Germany, Austria and Switzerland.**

**Thanks to ImranR98** and this outstanding idea! In my opinion, it contributes very well to the fact that the app strengthens the idea of FOSS/FLOSS and can make us all more independent from the data octopuses of this world.

Download: [240530_Obtainium_Wiki_GitHub de.pdf](https://github.com/ImranR98/Obtainium/files/15500418/240530_Obtainium_Wiki_GitHub.de.pdf)

**Comment by DwainZwerg** [comment]
@FreedomIsAChoice Magst du deine Übersetzung zu https://github.com/Hamster45105/Obtainium-Wiki/tree/i18n/pages hinzufügen? Es gibt jetzt nämlich ein Wiki, dass alle bearbeiten können.

**Reply by DwainZwerg** [reply]
@FreedomIsAChoice Falls dir nicht klar sein sollte, wie du die Übersetzung zu https://github.com/ImranR98/Obtainium-Wiki hinzufügst? Würdest du mir die Erlaubnis geben, deine Übersetzung hinzuzufügen?

**Reply by FreedomIsAChoice** [reply]
Hi @DwainZwerg,

danke für Deine Nachfrage. Du kannst meine Übersetzung gerne dem Wiki hinzufügen. Danke, dass Du mir die Arbeit abnimmst! 😅👍🏻
Über ein "Credits to..." würde ich mich freuen! 😉

Beste Grüße und gutes Gelingen!
FreedomIsAChoice
